config/config.exs

```diff
@@ -1,4 +1,4 @@
-use Mix.Config
+import Config
 
 case Mix.env() do
```

Comment:
I was waiting until we raise to Elixir 1.11 (I am following plug/ecto versions) so that we could use `config_env()` and eliminate `Mix` usage altogether. 

Wdyt?